integer/src/ubig.rs

```diff
@@ -71,7 +71,7 @@ pub struct UBig(pub(crate) Repr);
 impl UBig {
     /// Get the representation of UBig.
     #[inline]
-    pub(crate) fn repr(&self) -> TypedReprRef<'_> {
+    pub fn repr(&self) -> TypedReprRef<'_> {
```

Comment:
Where do you need to access this private representation? This type is not intended for outside use.